works.css

```diff
@@ -1,40 +1,123 @@
 body {
   background: #f3f3f3;
+  margin: 1px;
 }
 
 ul {
   list-style-type: none;
 }
 
+.nav-link-div {
+  display: flex;
+  margin: 1px;
+  padding-right: 1rem;
+  margin-right: 5rem;
+}
+
+.nav-link {
+  padding-right: 1rem;
+  padding-left: 1rem;
+  font-family: sans-serif;
+  color: #344563;
+}
+
 .grid-container {
   display: grid;
   grid-template-columns: auto;
-  grid-template-rows: auto auto auto;
+  grid-template-rows: auto auto auto auto;
+  justify-items: center;
 }
 
 .grid-card {
   background-color: white;
-  display: grid;
   gap: 1rem;
   border: 1px solid rgb(206, 218, 216);
   border-radius: 10px;
-  margin: 1rem;
   padding: 1rem;
   justify-content: center;
   align-content: center;
 }
 
-h1 {
-  font-family: sans-serif;
-  font-style: normal;
-  display: flex;
-  color: #172b4d;
+@media only screen and (max-width: 600px) {
+  .card-img {
+    width: 100%;
+    height: 100%;
+  }
+
+  .grid-card {
+    display: flex;
+    flex-direction: column;
+    margin: 1rem;
+  }
+
+  .contact-me {
+    padding: 2rem;
+  }
+
+  .about-me {
+    flex-direction: column;
+  }
+
+  .collaps {
+    display: none;
+  }
 }
 
-@media (max-width: 70%) {
+@media only screen and (min-width: 601px) {
```

Comment:
- [ ] Please follow the given requirements by the previous reviewer it will help to make your page responsive..desktop devices ranges between `992px` and `1100px` mostly laptop are `1024px`